mantidproject/mantid
# D33 add a wrapper component for all banks

**Description of work.**
This updates the D33 definition to encompass all detector banks in one component for convenience.

**To test:**

1. LoadEmptyInstrument
2. Open Instrument View
3. Go to Instrument Tab
4. In the component tree make sure that the 5 detectors are now inside one component called detector.

**Review and merge with PR [#111](https://github.com/mantidproject/mantidgeometry/pull/111) in `mantidgeometry`;  the IDF is generated with the script modified there.**



Fixes #23376 . 



**This does not require release notes, since the geometry is not changed, it's just a virtual component that is added for developers' well-being.**

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
